src/components/sbb-navigation-section/sbb-navigation-section.tsx

```diff
@@ -275,23 +266,14 @@ export class SbbNavigationSection implements ComponentInterface {
 
   // Set focus on the first focusable element.
   private _setNavigationSectionFocus(): void {
-    if (sbbInputModalityDetector.mostRecentModality === 'keyboard') {
-      getFirstFocusableElement(
-        Array.from(this._element.children).filter(
-          (e): e is HTMLElement => e instanceof window.HTMLElement,
-        ),
-      )?.focus();
-    } else {
-      // Focusing sbb-navigation__wrapper in order to provide a consistent behavior in Safari where else
-      // the focus-visible styles would be incorrectly applied
-      this._navigationSectionContainerElement.tabIndex = 0;
-      this._navigationSectionContainerElement.focus();
-
-      this._navigationSectionContainerElement.addEventListener(
-        'blur',
-        () => this._navigationSectionContainerElement.removeAttribute('tabindex'),
-        { once: true },
-      );
+    const firstFocusableElement = getFirstFocusableElement(
```

Comment:
Another thing I noticed is that previously on mobile the back button that appears was the first focusable to be focused but now it is the first label. Is this also ok?

Reply:
This happens on iOS with VoiceOver.

Reply:
I think it should be the back button, but I did not change this logic? :/ We would need to consider shadow dom as well here

Reply:
Yes, I was thinking the same thing. Because we didn't target the shadowDom before either, but somehow the back button was still focused by VoiceOver 🤔